src/Sorting/Model/Sortable.php

```diff
@@ -94,6 +94,35 @@ public function setSortingPositionAttribute($sortingPosition)
         $this->attributes[$this->sortingPositionColumn] = $sortingPosition;
     }
 
+    /**
```

Comment:
public function sortingPositionColumnValue():int
     {
         return (int) $this->{$this->sortingPositionColumn};
     }